lib/moab/storage_services.rb

```diff
@@ -13,81 +13,86 @@ module Moab
   # @note Copyright (c) 2012 by The Board of Trustees of the Leland Stanford Junior University.
   #   All rights reserved.  See {file:LICENSE.rdoc} for details.
   class StorageServices
-    # @return [StorageRepository] an instance of the interface to SDR storage
+    include Singleton
+
+    # this being a Singleton should result in only one class instance being created for
```

Comment:
I'm not sure this follows.   A singleton is about the object instance, not about the class instance. I'm don't believe`include Singleton` is doing anything here.

Reply:
yeah, agreed, i ended up thinking the same after discussion on slack week before last and seeing the new test continue to pass even when `include Singleton` was commented out.  pushed up another update removing the `include` and updating the comment to hopefully be more accurate.